### PR TITLE
VAN-4036 Added docker-scan  with trivy module

### PR DESCRIPTION
**Pull request by akashcldcvr**
Add Module to Scans docker image vulnerabilities using Trivy

### CODE REVIEW

File: pipeline-modules/app-service/azure/docker-scan-with-trivy.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      type: string
+      default: 'HIGH,CRITICAL'
+    trivy_args:
+      title: Extra Trivy Arguments
+      description: Arguments for Trivy command
+      type: array
+      default: []
+      items:
+        type: string
+    pool_name:
```
Comment (dshelley66) [inline pipeline-modules/app-service/azure/docker-scan-with-trivy.yaml:49]: why is this here as it isn't used?

File: pipeline-modules/app-service/azure/docker-scan-with-trivy.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      type: string
+      default: repo
+    trivy_version:
+      title: Trivy Version
+      description: Trivy Version in i.e- vX.X.X
+      type: string
+      default: v0.40.0
+    severity:
+      title: Severity
+      description: The severities (CRITICAL,HIGH,MEDIUM,LOW,UNKNOWN) to include in the scan (comma sperated). Defaults to CRITICAL,HIGH,MEDIUM,LOW,UNKNOWN
```
[the remark this answers is not in the document]
Comment (akashcldcvr) [reply pipeline-modules/app-service/azure/docker-scan-with-trivy.yaml:39]: Updated

File: pipeline-modules/app-service/azure/docker-scan-with-trivy.yaml (hunk 1 above)
Comment (akashcldcvr) [reply pipeline-modules/app-service/azure/docker-scan-with-trivy.yaml]: removed
